src/js/languages/default.json

```
@@ -77,13 +77,17 @@
     "dropPromptMultiple": "Drop files here or",
     "files": "files",
     "maxFile": "Attach a maximum of {max} files only.",
+    "remove": "remove file: {fileName}",
+    "removeAll": "remove all",
     "maxSizeSingle": "The file is too large. Select a file no larger than {maxSize}.",
     "maxSizeMultiple": {
       "singular": "One file is too large. Select files which are no larger than {maxSize}.",
       "plural": "{numOfFiles} files are too large. Select files which are no larger than {maxSize}."
     },
-    "remove": "remove",
-    "removeAll": "remove all"
+    "alert": {
+      "maxSize": "Error - remove file: {fileName} to continue. This file is too large. Attach files which are no larger than {maxSize}.",
+      "maxFile": "Error - remove file: {fileName} to continue.  Attach a maximum of {max} files only."
+    }
```

same. I still think we need to avoid saying "Error - remove file"
```suggestion
    "alert": {
      "maxSize": "File {fileName} is too large. Attach files which are no larger than {maxSize}.",
      "maxFile": "Attach a maximum of {max} files only. File {fileName} is beyond this limit"
    }
```

---

@MikeKingdom I do get your comment about using the word `error` Bill just mentioned that its nice to know that something is wrong right away instead of listening for the screen reader to finish the filename (especially if its long) to then say the file was to large. So I wonder if we can do something like 

`This file {name} is too large to upload. Remove it to continue.`
`File is too large. 'filename.pdf' must be removed to continue.`

---

> @MikeKingdom I do get your comment about using the word `error` Bill just mentioned that its nice to know that something is wrong right away instead of listening for the screen reader to finish the filename (especially if its long) to then say the file was to large. So I wonder if we can do something like
> 
> `This file {name} is too large to upload. Remove it to continue.` `File is too large. 'filename.pdf' must be removed to continue.`

This is more like how we've been asked to word things from a "tone" perspective. We used to have someone that focused on tone.

I'll let you folks work out what works best for Bill since he's obviously a better authority on that.